### ISSUE TITLE
Formatting of the manual

### ISSUE THREAD

**Issue by leifeld** [leifeld/dna#113]
In Chapters 4-8, the following changes need to be made:

1. Urgent: Update the export chapter 7 based on the new contents of the algorithm chapter 2. Ensure that all calculations are correct.
2. References like `see Section 3` need to changed from italic to normal font, and "Section" needs to be replaced by "Chapter" if the highest TOC level is affected.
3. There are many text portions that are emphasised using bold text or quotation marks. If this is code or something similar (e.g., button labels, file names etc.), it should instead be set using the `\code` macro. If this is just some emphasised text, this should be typeset using `\emph`. Do not use `\textit`.
4. Long tables with definitions should be replaced by `description` environments. See the example in the "User Management: Multiple Coders and Permissions" section (lines 1057 ff.).
5. Items in an `itemize` environment should not use curly braces. This clutters the code unnecessarily. See my examples in Chapter 3 for best practice.
6. If there are screenshots that do not have a border, add one using the `frame` argument to `\includegraphics`. I have done this in two or three instances in the installation chapter, for your reference.
7. Typeset URLs using the `\url` command if the label of the URL that is displayed in the text does not deviate too much from the link text.

**Comment by leifeld** [comment]
Thanks.

Regarding point 2: I can still some some instances where this happens. For example in Section 4.1.2, "Creating and Using a Remote Database (MySQL)" and in Section 4.2. Perhaps do a keyword search for `\textit`?

Regarding point 3: I can still see some instances where text is bold. For example throughout Sections 4.3 and 4.4.

About point 4: I just replaced an ``addmargin`` environment by a `description` environment in Section 4.1.2, but I haven't seen any other occurrences of that. There are some instances in which plain `itemize` environments were misused to typeset what ought to be `description` environments, for example in Section 7.6, "Normalization". Please fix. Thanks.

Regarding point 6: You are right; do not add the frame in these instances.

**Comment by leifeld** [comment]
Another minor thing I noticed: typeset something like `organizations x concept` as `organizations $\times$ concept`.

**Comment by leifeld** [comment]
Another formatting issue: can you please typeset the document such that we have one sentence per line? That will permit more straightforward version control. I have done this for chapters 1-3 already.

**Comment by JBGruber** [comment]
`\textit`: Thought I had already searched for that. None are left now.

Regarding point 3: Ok, I did not think you wanted bold completely gone as well. Only very few special cases are left now where bold makes more sense than \emph.

`addmargin`: I don't remember why that was there. I think it was in the original markdown file. 

About point 4:  I replaced itemize with description environments where this makes sense

`$\times$`: Done. This does not work inside the table captions in chapter 7 though.

One sentence per line: Suddenly this makes sense to me! I used regex to replace ". " with ".\n" where this made sense.

This should be it.
